main/src/library/pkger/Manifest.flix

```diff
@@ -0,0 +1,287 @@
+namespace Manifest {
+    pub enum License {
+        case MIT
+        case GPLv3
+        case Apache2
+        case BSD
+        case Other(String)
+    }
+
+    pub type alias Author = { name: String
+                            // Is there an email datatype, or is it just commonly a String?
+                            , email: String
+                            }
+
+    pub type alias Package = { name: String
```

Comment:
What exactly is a package name? Just a string?

Reply:
Yes, that was my thinking. Ideally I'd like to limit the name to being a valid Flix identifier, as a deeper integration into the language could be useful.